src/Rotations.jl

```diff
@@ -1,5 +1,5 @@
 # we generate code in this module, so precompile where possible
-#__precompile__(true)
+__precompile__(true)
```

Comment:
Quaternions is no longer a dependency. I was surprised this has been omitted. Oops.